Common/Net/Resolve.cpp

```diff
@@ -269,7 +269,7 @@ int inet_pton(int af, const char* src, void* dst)
 			else if (ch < '0' || ch > '9')
 				return 0;
 			else
-				x = x * 10 + ch - '0';
+				x *= 10 + ch - '0';
```

Comment:
Come on now. This is just wrong.

This means `x = x * (10 + ch - '0')`, not `x = x * 10 + ch - '0';`

Again, please test your changes.